lib/Controller/ApiController.php

```diff
@@ -1180,7 +1180,7 @@ public function insertSubmission(int $formId, array $answers, string $shareHash
 		$this->formsService->setLastUpdatedTimestamp($formId);
 
 		//Create Activity
-		$this->formsService->notifyNewSubmission($form, $submission->getUserId());
+		$this->formsService->notifyNewSubmission($form, $submission->getUserId(), $submission);
```

Comment:
Duplicated information. If the whole submission is passed then we do not need the userid.
```suggestion
		$this->formsService->notifyNewSubmission($form, $submission);
```

So the `notifyNewSubmission` needs of course to be adjusted